backend_jobs/recognition_pipeline/filters/filter_by_resolution.py

```diff
@@ -40,6 +42,9 @@ def is_supported(self, image):
         min_width = self.prerequisites['width']
         if image_attribute['width'] >= min_width and \
           image_attribute['height'] >= min_height:
+            # current url is in a supported resolution
+            image[database_schema.COLLECTION_IMAGES_FIELD_URL_FOR_RECOGNITION_API] =\
```

Comment:
nit: we're not storing this in database, so we should not have this constant in the schema.

Reply:
Yes that's right. Added a new constants.py file to the recognition_pipeline.pipeline_lib with this field. Can you think of any place better for it?